doc/concept/overview.md

```diff
@@ -0,0 +1,78 @@
+
+1. A producer of energy requests a certificate for each MWh of energy produced, and the national issuing body for GO's, compliant with the central authority (AIB), issues the GO(s) to the producer
+2. The GO(s) is/are sold to a supplier
+3. The supplier sells the GO(s) to a consumer
+4. The consumer can then "use" the GO, by [cancelling](https://en.energinet.dk/Energy-data/Guarantees-of-origin-el-gas-hydrogen/#accordion-cancellation), effectively attributing consumption of the renewable energy amount specified by the GO to the consumer.
+
+To summarize, the existing GO system is based on the following principles:
+- Yearly certification
+- Volume based accounting
+- Exchange of certificates between parties through a centralised system 
+- The certificate is a tradeable asset
+- There is no requirement for a direct physical link between the certificate and the energy supplied
+
+#### Drawbacks
+Despite the good concepts behind the GO certification system, it suffers from a number of serious drawbacks. One of these is the very low certificate prices, making it [difficult to generate enough economic incentives](https://www.sciencedirect.com/science/article/abs/pii/S0301421504002423) for improvements and investments in renewable energy production facilities. Another drawback is the property of not to link any physical properties to the certification system, which causes [neglection of transportation costs and over-exploitation of interconnector capacities](https://www.sciencedirect.com/science/article/abs/pii/S0301421516302889). Finally, there have been cases of countries with no physical interconnections [trading GOs](https://akjournals.com/view/journals/204/41/4/article-p487.xml?body=contentsummary-24716). These drawbacks ultimately hinder a trustworthy, transparent documentation of energy consumption and unfortunately don't create the positive impact on further renewable energy investments, as intended.  
+
+
+## Challenges in the current and future energy grid
+At times, certain grid areas experience a supply by renewables that is [bigger than the grid capacity](https://www.caiso.com/documents/curtailmentfastfacts.pdf), which brings a number of [challenges to the existing, physical grid](https://www.rff.org/publications/explainers/renewables-101-integrating-renewables/): 
+
+- It becomes [difficult to stabilize the grid frequency](https://www.engineering.com/story/grid-frequency-stability-and-renewable-power) in the electricity grid.
+- Insufficient capacity of the existing grid creates [bottlenecks](https://www.zerohedge.com/energy/grid-bottlenecks-could-derail-europes-renewable-energy-boom), limiting transportation between certain areas 
+- [Less centralized load balancing](https://research.rug.nl/en/publications/local-balancing-of-the-electricity-grid-in-a-renewable-municipali) might be necessary 
+- Makes it hard to validate the actual energy supplied to power-to-x applications and the origin of the losses in the process
```

Comment:
Adding comment here again, as it disappeared. Not sure what "origin of the losses" refer to.

Reply:
It is addressing whether the loss is from a renewable energy source or from conventional energy sources. This is significant when considering hydrolyzers they have a 40% conversion loss.

Reply:
```suggestion
- Makes it hard to validate the origin of the actual energy supplied to power-to-x applications, especially given the losses in the conversion process
```